velox/functions/prestosql/tests/InPredicateTest.cpp

```diff
@@ -132,6 +132,15 @@ class InPredicateTest : public FunctionBaseTest {
     rowVector = makeRowVector({dict});
     result = evaluate<SimpleVector<bool>>("c0 IN (2, 5, 9)", rowVector);
     assertEqualVectors(expected, result);
+
+    // an in list with nulls only is always null.
+    result = evaluate<SimpleVector<bool>>("c0 IN (null)", rowVector);
+    expected = makeFlatVector<bool>(
```

Comment:
nit: it would be more readable to use BaseVector::createNullConstant(BOOLEAN(), size, pool());